gtsam_unstable/slam/ReadMe.md

```diff
@@ -0,0 +1,39 @@
+# SLAM Factors
+
+## SmartFactors
+
+These are "structure-less" factors, i.e., rather than introducing a new variable for an observed 3D point or landmark, a single factor is created that provides a multi-view constraint on several poses and/or cameras.
+
+### SmartRangeFactor
+
+An experiment in creating a structure-less 2D range-SLAM factor with range-only measurements.
+It uses a sophisticated `triangulate` logic based on circle intersections.
+
+### SmartStereoProjectionFactor
+
+Version of `SmartProjectionFactor` for stereo observations, specializes SmartFactorBase for `CAMERA == StereoCamera`.
```

Comment:
comment at the top of SmartStereoProjectionFactor says calibration is fixed, but that seems strange since the factor takes variables of type `CAMERA' in values.

Reply:
Yeah, I checked and StereoCamera has no facilities to update the calibration. Maybe this is why the derived classes are weird...